core-web/libs/block-editor/src/lib/nodes/ai-content/ai-content.node.ts

```diff
@@ -0,0 +1,75 @@
+import { Node } from '@tiptap/core';
+declare module '@tiptap/core' {
+    interface Commands<ReturnType> {
+        AIContent: {
+            addHelloWorld: () => ReturnType;
+        };
+    }
+}
+export const AIContent = Node.create({
+    name: 'aiContent',
+    apiKey: '',
+    description: 'AI Content',
+    addAttributes() {
+        return {};
+    },
+    parseHTML() {
+        return [
+            {
+                tag: 'div'
+            }
+        ];
+    },
+    renderHTML() {
+        return ['div'];
+    },
+    addOptions() {
+        return {
+            inline: false
+        };
+    },
+    inline() {
+        return this.options.inline;
+    },
+    group() {
+        return 'block';
+    },
+    addCommands() {
+        return {
+            ...this.parent?.(),
+            addHelloWorld:
+                () =>
+                ({ commands }) => {
+                    return commands.insertContent({ type: this.name });
+                }
+        };
+    },
+    addNodeView() {
+        return () => {
+            const dom = document.createElement('div');
+
+            const input = document.createElement('input');
+            input.type = 'text';
+            input.placeholder = 'Ask AI to write something';
+            input.style.width = '100%';
+            input.style.outline = 'none';
+            input.style.color = '#6C7389';
+            input.style.fontFamily = 'Assistant';
```

Comment:
We are not using this font for now, it is implemented in the design, but not in the code. As Rafael mentioned we have variables for this, so later we will change the variable for the font to use `Assistant`